protocol/storage/miner.go

```diff
@@ -916,7 +915,7 @@ func (sm *Miner) submitPoSt(start, end *types.BlockHeight, seed types.PoStChalle
 
 // Query responds to a query for the proposal referenced by the given cid
 func (sm *Miner) Query(c cid.Cid) *storagedeal.Response {
-	storageDeal := sm.porcelainAPI.DealGet(c)
+	storageDeal := sm.porcelainAPI.DealGet(context.TODO(), c)
```

Comment:
`Miner#Query` should be modified to accept the context created in `Miner#handleQueryDeal`.

Reply:
Do you mean you want me to create a context in `handleQueryDeal`? Because there isn't currently a context in that method.